style.css

```diff
@@ -634,3 +633,390 @@ footer .footer-icons > li {
   opacity: 1;
   width: inherit;
 }
+
+@media (min-width: 992px) {
+  body {
+    overflow-x: hidden !important;
+  }
+
+  /* header section */
+  header {
+    padding: 10px 10%;
+  }
+
+  .hide-on-large {
+    opacity: 0;
+  }
+
+  .show-on-large {
+    display: flex;
+  }
+
+  .mx-20 {
+    margin: 0%;
+  }
+
+  .headline-section {
+    height: 100vh;
+    margin-bottom: -1.9%;
+    justify-content: space-between;
+    background-image: none;
+  }
+
+  .headline-text-contents {
+    margin: 0 0 0 5%;
+    width: 50%;
+  }
+
+  .headline-text-contents > h1 {
+    font-size: 72px;
+    line-height: 81.5px;
+  }
+
+  .headline-text-contents > h2 {
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .headline-text-contents > p {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .desktop-headline-bg-section {
+    margin-top: 0%;
+    height: 100%;
+    width: 80%;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .desktop-headline-bg-container {
+    margin-top: -17.2%;
+    height: 80%;
+    width: 100%;
+  }
+
+  .desktop-headline-bg {
+    height: 130%;
+    width: 100%;
+  }
+
+  .sider-menu {
+    width: 150px;
+    height: 100%;
+    justify-content: center;
+  }
+
+  .sider-menu > ul {
+    align-items: center;
+    flex-direction: column;
+    justify-content: center;
+  }
+
+  .sider-menu > ul > li {
+    padding: 20px 0;
+    margin: 0 10px 0 -30px;
+  }
+
+  .sider-menu > ul > li a:hover {
+    background-color: #ff6b00;
+    padding: 10px;
+    border-radius: 100%;
+    transition: all 0.5s ease-in-out;
+  }
+
+  /* work section */
+  .works-section {
+    padding: 10px 5%;
+  }
+
+  .heading-recent-works {
+    font-size: 40px;
+    padding-top: 2%;
+  }
+
+  .works-section .ruler {
+    width: 75%;
+    height: 4px;
+    margin: 3% 0 0 4%;
+    background-color: #212121;
+    border-radius: 15%;
+  }
+
+  .works-section > hr {
+    display: none;
+  }
+
+  /* work section multi-post container */
+  .multi-post-container {
+    flex-direction: row;
+    align-items: flex-start !important;
+  }
+
+  .multi-post-container > img {
+    height: auto;
+    width: 55%;
+  }
+
+  .multi-post-content {
```

Comment:
Please add an active image for the project section as it's broken. Kindly see the screenshot below for reference.
![project](https://user-images.githubusercontent.com/35504149/135510692-b2eec310-6aa2-44d1-97ea-21e6eabc2be8.png)